modules/constants.js

```diff
@@ -223,6 +223,7 @@ exports.ERROR_TYPE = {
     FAILED_COMMAND_ERROR: 'FailedCommandError',
     UPDATE_INITIALIZATION_ERROR: 'UpdateInitializationError',
     DATA_MODULE_INITIALIZATION_ERROR: 'DataModuleInitializationError',
+    OPERATIONALDB_MODULE_INITIALIZATION_ERROR: 'OperationalDbModuleInitializationError',
```

Comment:
I have encountered a good approach in libp2p for handling errors in general, let's try to review it